protocol/activity_center_persistence_test.go

```diff
@@ -205,17 +205,22 @@ func (s *ActivityCenterPersistenceTestSuite) Test_DeleteActivityCenterNotificati
 	}
 
 	// Test: soft delete the notifications that have Message.ID == messages[1].ID
-	// or LastMessage.ID == chat.LastMessage.
 	_, err = p.DeleteActivityCenterNotificationForMessage(chat.ID, messages[1].ID, currentMilliseconds())
 	s.Require().NoError(err)
 
-	for _, id := range []types.HexBytes{nID2, nID3} {
-		notif, err = p.GetActivityCenterNotificationByID(id)
-		s.Require().NoError(err, notif.ID)
-		s.Require().True(notif.Deleted, notif.ID)
-		s.Require().True(notif.Dismissed, notif.ID)
-		s.Require().True(notif.Read, notif.ID)
-	}
+	notif, err = p.GetActivityCenterNotificationByID(nID2)
```

Comment:
Also add this to prevent potential `panic` on access to `nil` pointer:
```go
s.Require.NotNil(notif)
```

Reply:
In fact, when `err != ni` , `notif` will be `nil`.
So this line is not quite good:
```go
s.Require().NoError(err, notif.ID)
```